src/components/container/FeaturedJobs/FeaturedJobs.js

```diff
@@ -31,6 +31,12 @@ function FeaturedJobs() {
             />
           ))}
         </div>
+        <a className="FeaturedJobs__viewMoreLink__bottom" href="/#">
```

Comment:
would be cleaner with the modifierm´--top and --bottom, but that's fine!

Reply:
agreed

Reply:
Okay, I'll get to it